Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -137,6 +137,260 @@ Once the permission `START_MAIN_ACTIVITY` has been created, apps can request it
 
 We recommend using a reverse-domain annotation when registering a permission, as in the example above (e.g. `com.domain.application.permission`) in order to avoid collisions with other applications.
 
+## Injection Flaws
+
+[Injection Flaws](https://owasp.org/www-community/Injection_Flaws "Injection Flaws") allow attackers to leverage the interconnection of applications and pass malicious code between the affected application and either another system (such as a server) or another connected client.
+
+### Deep Links
+
+Android apps can expose functionality through deep links (which are a part of Intents). They can expose functionality to:
+
+- other apps (via deep links or other IPC mechanisms, such as Intents or BroadcastReceivers).
+- the user (via the user interface).
+
+None of the input from these sources can be trusted; it must be validated and/or sanitized. Validation ensures processing of data that the app is expecting only. If validation is not enforced, any input can be sent to the app, which may allow an attacker or malicious app to exploit app functionality.
+
+## Fragment Injection
```

Comment:
Let's move this back to the test since this is a very specific risk (and in the near future we'll actually move to the CODE category).